support/support.adoc

```diff
@@ -0,0 +1,73 @@
+= Support
+:icons: font
+:sectanchors: 
+:url-repo: https://github.com/AdoptOpenJDK/website-adoptium-documentation
+
+== Community Support
+Read https://medium.com/@javachampions/java-is-still-free-2-0-0-6b9aa8d6d244[Java Is Still Free (2.0.0)] for some 
+background information about Java support options. We are prepared to  stand behind our release quality binaries, 
+so each build that is identified as a release receives support via the Adoptium community. Our support means that 
+you can raise an issue to describe a bug you have found in the build, and we will work with you and the appropriate 
+development team to resolve it. Any fixes we identify will be delivered as part of the next Adoptium release.
+
+As a community of open source developers, our commitment is to triage any issues raised and champion them in the appropriate 
+source code project. Of course, if the problem arises from the way we build and test the code we can fix that directly. 
+For higher levels of assurance you should contact commercial companies offering support.
+
+== Commercial Support
```

Comment:
Since we do not have commercial support at the moment the header should be removed.

`Release Roadmap` is on same level as `Community Support`

Reply:
Done